### PR TITLE
Create PAC and party datatable

### LINKED ISSUE #4651: Build minimal PAC and party committees total datatable
### Summary

**What we're after:**
_Build a minimal PAC and party committees totals datatable using /totals/pac-party/ API call with these search filters committee name/ID and 2-year period._

### Completion criteria

- [x] PAC and party committee datatable is created under a feature flag

### PR DESCRIPTION
**Pull request by patphongs**
## Summary (required)

- Resolves #4651 

Create new PAC and party datatable with committee name/ID and time period filters.

**Note:** This datatable is under a feature flag. There are also filters and fields that were added prior to the API work being done. I've noted the items as `TODO` in the code, so we know where to revisit when that data is in the API.

### Required reviewers

1 UX and 1 front-end

## Impacted areas of the application

General components of the application that this PR will affect:

-  PAC and party datatable

## Screenshots

Screen Shot 2021-05-24 at 5 45 02 PM

Screen Shot 2021-06-03 at 1 28 06 PM

Screen Shot 2021-06-03 at 1 25 34 PM

## How to test

- `npm run build`
- ./manage.py runserver
- Go to datatable: http://localhost:8000/data/committees/pac-party/. Click on the details panel to see more info.
- Go to committees browse data page to see the link to the datatable: http://localhost:8000/data/browse-data/?tab=committees

### CODE REVIEW

**Comment by rfultz** [comment]
The main nav has changed for me in the medium widths
image

File: fec/data/constants.py [hunk 1]
```diff
@@ -382,6 +382,7 @@
     ('candidates-office-senate', ['Name', 'Party', 'State', 'Receipts', 'Disbursements']),
     ('candidates-office-house', ['Name', 'Party', 'State', 'District', 'Receipts', 'Disbursements']),
     ('committees', ['Name', 'Committee ID', 'Treasurer', 'Type', 'Designation', 'First filing date']),
+    ('pac-party', ['Name', 'Type', 'Receipts', 'Disbursements', 'Cash on  hand']),
```
Comment (rfultz) [inline fec/data/constants.py:385]: Does the double space in "Cash on  hand" matter?

Comment (patphongs) [reply fec/data/constants.py]: removed the extra space, thanks!

**Review by rfultz — APPROVED** [review]

**Review by JonellaCulmer — APPROVED** [review]
So far so good! Thanks, @patphongs

**Pull request merged**